assemble/pom.xml

```diff
@@ -72,7 +73,7 @@
                         <id>make-assembly</id>
                         <phase>package</phase>
                         <goals>
-                            <goal>attached</goal>
+                            <goal>single</goal>
```

Comment:
what does it have to change?

Reply:
If you read the commit message fully it says this :
```
- Use goals `single` instead of `attached` by the recommendation at
http://maven.apache.org/plugins-archives/maven-assembly-plugin-2.5.5/attached-mojo.html
```
If you check the page it says attached has been deprecated for a while now (this page is for version 2.5.5) and it suggests using single. And when I upgraded to the latest version of the plugin, attached has been removed, thus crashing the compilation.

Clear now? :)

Reply:
sure, thanks